### PR TITLE
Some fixes

### LINKED ISSUE #6: Return type of webhook option is not type object
## Describe the bug

> An error occurred while returning an empty webhook.

### Expected behaviour:

> When accessing api, an error _500 (Internal Server Error)_ occurred, because it was waiting to receive an **object**, and not an **empty array**.

### Actual behaviour:

> After correction, the answer is always of type **object**.

### Steps to reproduce the behavior:

> Сode example:
>
> ```php
> $options = [
>            'webhook' => array_filter([
>                'on_complete' => $config->get("$root.webhooks.on.complete"),
>                'on_update'   => $config->get("$root.webhooks.on.update"),
>            ]),
>        ];
>
> return $options;
>
> Request to the B2B API resulted in a `500 Server Error` response:
> {"state":"fail","stamp":"2018-11-14T05:23:08.920Z","event":{"uid":"","stamp":"2018-11-14T05:23:08.919Z","cls":"Global"," (truncated...)

## System information

> Please, complete the following information:

 - PHP version: `7.2.11`
 - Package version: `2.3.0`

## Additional context

> Fixed tests.

### PR DESCRIPTION
**Pull request by eldario**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | No

## Description

Changed the return type of webhook. 

Fixes #6

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I wrote unit tests for my code
- [x] I have made changes in [CHANGELOG.md](https://github.com/avto-dev/b2b-api-php-laravel/blob/master/CHANGELOG.md) file

### CODE REVIEW

**Comment by tarampampam** [comment]
@eldario Updated changeslog is required

**Comment by tarampampam** [comment]
Also - make an issue first (and link issue with entry into changeslog file)

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog] and this project adheres to [Semantic Versioning].
 
+## [v2.4.0] - 2018-11-13
```
Comment (tarampampam) [inline CHANGELOG.md:7]: Где сноска ссылки `[v2.4.0]`?

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Comment by tarampampam** [comment]
Update PR description (link to the issue) and check checkboxes statuses

File: CHANGELOG.md (hunk 1 above)
Comment (eldario) [reply CHANGELOG.md:7]: Исправил

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
All looks file! Thx for your PR!

**Pull request merged**